.github/workflows/test-node.yml

```diff
@@ -24,6 +24,12 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - run: npm ci
-      - run: npm run test:node
+      - run: npm run test:node:junit
         env:
           CI: true
+      - name: Upload test results
+        if: always()
+        uses: ably-labs/test-observability-action@main
+        with:
+          server-url: 'https://test-observability.herokuapp.com'
+          server-auth: ${{ secrets.TEST_OBSERVABILITY_SERVER_AUTH_KEY }}
```

Comment:
Just a note that this now exists as an org-wide secret (see https://github.com/ably/test-observability-action/pull/9), so if it's a repo-level secret here it can now be removed.